### DISCUSSION TITLE
New social record

### DISCUSSION THREAD

**Discussion by Group24Steven** [withastro/starlight#1533] [Q&A] [answered]
### What version of `starlight` are you using?

0.19.0

### What is your idea?

I'd like to enquire a new social(icon) being xing.
Xing is just a company with headquarters in germany for jobs I think.
I need this social to include into my docs page as a socialicon.
Would be available then as a new record inside starlight config just called `xing` (Xing in lowercase obviously...)

Thanks in advance for adding it. 😌

### Why is this feature necessary?

Extends social-records. A workaround this would be to manipulate the already existing socials so that it's not being a specific record like previously but a custom social which could store a custom icon coming from any source or icon libraries such as fontawesome.

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by at-the-vr** [comment]
Linking the Discord [Workaround](https://ptb.discord.com/channels/830184174198718474/1070481941863878697/1209081001809027153) suggested by Chris, applicable on any social icon (works well with https://icones.js.org/)

**Comment by HiDeoo** [comment, marked as answer]
Also sharing a workaround [note](https://hideoo.dev/notes/starlight-custom-social-links) I wrote a long time ago about adding custom social links to Starlight.
